project_task_reply_stage/models/mail_message.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailMessage(models.Model):
+    _inherit = "mail.message"
+
+    @api.model_create_multi
+    def create(self, values_list):
+        messages = super(MailMessage, self).create(values_list)
+        for res in messages:
```

Comment:
```suggestion
        for message in messages:
```
`res` should be used for returned values/records from a method.  It's weird to use it in the manner you do.